documentation/source/building.rst

```diff
@@ -207,6 +207,16 @@ Environment Variables
     You can usually write out such a file from the simulator's GUI.
     This is currently supported for the Mentor Questa, Mentor ModelSim and Aldec Riviera simulators.
 
+.. envvar:: COCOTB_ENTRY
+
+        The Python framework entry point from the GPI.
```

Comment:
Could you also add a line like ``COCOTB_ENTRY      Python entry point (default: cocotb._initialize_testbench)`` to https://github.com/cocotb/cocotb/blob/master/cocotb/share/makefiles/Makefile.sim#L76 ?

Reply:
Done.